helm/hpcc/templates/_helpers.tpl

```diff
@@ -68,19 +68,49 @@ Translate a port list to a comma-separated list
  {{- end -}}
 {{- end -}}
 
+{{- define "hpcc.getFirstPlaneForLabel" -}}
+{{- $root := .root -}}
+{{- $label := .label -}}
+{{- $storage := ($root.Values.storage | default dict) -}}
+{{- $planes := ($storage.planes | default list) -}}
+{{- $firstPlane := dict -}}
+{{- range $plane := $planes -}}
+{{- if not $firstPlane.plane -}}
+{{- $labels := $plane.labels | default (list "data") -}}
+{{- if (has $label $labels) -}}
+{{- $_ := set $firstPlane "plane" $plane.name -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+{{- if $firstPlane.plane -}}
+{{- $firstPlane.plane -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "hpcc.hasPlaneForLabel" -}}
+{{- if (include "hpcc.getFirstPlaneForLabel" .) -}}
+true
```

Comment:
I think this will end up as a string rather than a boolean value if called via include, as you do elsewhere...